### DISCUSSION TITLE
Sample Management

### DISCUSSION THREAD

**Discussion by Maxklui** [bangcorrupt/hacktribe#163] [Q&A] [answered]
Hi

I apologize for posting something else but this has been on my heart for a long time

I have the following problem when I categorize my samples. In the program, I usually always enter the category what the sample. If I import it then the samples are shown to me somehow. In the Google Drive link is a video showing what I mean:

The screenshot shows how I normally categorize them:
![image](https://user-images.githubusercontent.com/97652803/168057759-6cb832eb-4e1e-466c-8178-27d5c0fdc9d6.png)

https://drive.google.com/file/d/12dyYjofsZq7hkrfnLAvIVZsrahQEjJob/view?usp=sharing

I would be so grateful if someone could tell me what I'm doing wrong. Thanks in advance

**Comment by Piergap** [comment]
I think i know this user :-)

**Comment by BKLronin** [comment, marked as answer]
Unfortunately you have to order your samples right in this editor with the buttons on the left. The tags are more a kind of filter. If add new samples you need all the previous samples.

**Reply by Maxklui** [reply]
@BKLronin meinst du die runden button ganz links?

**Reply by BKLronin** [reply]
[]
![Those](https://user-images.githubusercontent.com/6392076/168091341-5e53840f-3d66-41e7-94c5-708098e90dc5.png)
(

**Reply by Maxklui** [reply]
@BKLronin Denn ich möchte nämlich mit Shift und drehen des Oszillatorknopfs durch die Samplearten durchswitchen. Ist so was möglich?

**Reply by BKLronin** [reply]
Yes, once you sorted all the samples BD SD HH all in groups close together, you can then switch with shift and encoder to the next group. Like from BD so SD without the need to scroll individually.
